modules/gmc-agent/src/main/java/org/gridgain/action/ActionMethod.java

```diff
@@ -30,15 +30,19 @@
     /** Controller class. */
     private final Class<?> controllerCls;
 
+    /** Is need auth. */
+    private final boolean isNeedAuth;
```

Comment:
Why we need this flag? As far as I remember only one action does not require auth: "/authenticate".
All other - require. May be we can check for action name instead of this flag?

Reply:
Fix.